packages/components/src/color-picker/index.native.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * External dependencies
+ */
+import { View, Text, TouchableWithoutFeedback, Platform } from 'react-native';
+import HsvColorPicker from 'react-native-hsv-color-picker';
+import tinycolor from 'tinycolor2';
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { BottomSheet } from '@wordpress/components';
+import { withPreferredColorScheme } from '@wordpress/compose';
+import { Icon, check, close } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+function ColorPicker( {
+	shouldEnableBottomSheetScroll,
+	shouldDisableBottomSheetMaxHeight,
+	isBottomSheetContentScrolling,
+	setColor,
+	activeColor,
+	onNavigationBack,
+	onCloseBottomSheet,
+	getStylesFromColorScheme,
+} ) {
+	const isIOS = Platform.OS === 'ios';
+	const isGradient = activeColor?.includes( 'linear-gradient' );
```

Comment:
I think it's already checked and could be passed as a prop

Reply:
Done!